deploy/crds/planetscale_v2_vitesscluster_crd.yaml

```diff
@@ -781,6 +791,19 @@ spec:
                                           items:
                                             type: object
                                           type: array
+                                        extraVitessFlags:
+                                          additionalProperties:
+                                            type: string
+                                          description: 'ExtraFlags can optionally
+                                            be used to override default flags set
+                                            by the operator, or pass additional flags
+                                            to child components. All entries must
```

Comment:
It will be good to make the documentation uniform across all CRDs. Here we have `child components`, elsewhere we have `all child components` or `Vitess components`.

Reply:
I can try to make it more uniform, but the field is somewhat different depending on where it is.  The idea is that there's a single parent `ExtraVitessFlags` in `VitessClusterSpec` which gets copied to where those flags need to be available.  Currently they need to be made available to `vtgate`, `vtctld` and `vttablet`. When it exists in a parent accessible by one of those child components, it has a more specific use case.  I suppose we could generally say that regardless the flags are accessible to `all child components` and have a uniform comment.

Reply:
I suggest we only document the field in VitessCluster. The other fields are all meant to be for internal use only; users can't even set them because they aren't present in the `*Template`. For these kinds of internal fields, I generally write the doc comment assuming the audience is other vitess-operator contributors. In this case, all they need to know is, "this field is just a copy of the one from VitessCluster; go look there to see what this does."

Reply:
Adjusted per @enisoc's suggestion https://github.com/planetscale/vitess-operator/pull/14/commits/226927d4cabfc3a79db1fcea1f2079d61b4ec302.